src/components/ModelTableList/ModelSummary/ModelSummary.test.tsx

```diff
@@ -23,8 +23,8 @@ describe("ModelSummary", () => {
         }),
       },
       machines: {
-        "0": machineChangeDeltaFactory.build(),
-        "1": machineChangeDeltaFactory.build(),
+        "0": modelDataMachineFactory.build(),
+        "1": modelDataMachineFactory.build(),
```

Comment:
We were using the wrong type of factories for a bunch of these tests but now we get warned if they're wrong.